imgix.go

```diff
@@ -153,11 +154,22 @@ func (c *Client) PathWithParams(imgPath string, params url.Values) string {
 
 	urlString += imgPath
 
+	for key, val := range params {
+		if strings.HasSuffix(key, "64") {
+			valData := []byte(val[0])
+			base64EncodedVal := base64.URLEncoding.EncodeToString(valData)
+			base64EncodedVal = strings.Replace(base64EncodedVal, "=", "", -1)
+			params.Set(key, base64EncodedVal)
+		}
+	}
```

Comment:
Sure thing, I can move it.

Our new Base64 variant params are designed to make advanced usage of imgix easier. They work especially well with `blend`, `mark`, and `txt` params that are nested multiple times.